tests/spec/date_series_ops/test_date_comparisons.py

```diff
@@ -129,3 +131,26 @@ def test_is_equal_with_str_date(spec_test):
             4: None,
         },
     )
+
+
+def test_is_in_dates(spec_test):
+    dates = [date(2010, 1, 1), date(2010, 1, 2)]
```

Comment:
I wonder if it's worth making the target dates here span one of the example dates to make it clear that the semantics are "exact match to a date in the list" rather than "within the range implied by the list members" e.g. if you used `1990-01-01` and `2010-01-01`.